sentry-kubernetes.py

```diff
@@ -16,12 +16,12 @@
 # mapping from k8s event types to event levels
 LEVEL_MAPPING = {"normal": "info"}
 
-LOG_LEVEL = os.environ.get("LOG_LEVEL", "error")
+LOG_LEVEL = os.environ.get("LOG_LEVEL", "info")
```

Comment:
Why this change? I suppose this will produce a lot of noise.

Reply:
LOG_LEVEL is being used to set the logging log level for this script [here](https://github.com/getsentry/sentry-kubernetes/blob/master/sentry-kubernetes.py#L54). Setting it to Warning makes the container report nothing to stdout when initializing.